cves/CVE-2019-6975.yml

```diff
@@ -66,18 +69,24 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
 - commit: 40cd19055773705301c3428ed5e08a036d2091f3
-  note:
+  note: |
+    (2019-02-11) Backport into 2.1.x.
 - commit: 1f42f82566c9d2d73aff1c42790d6b1b243f7676
-  note:
+  note: |
+    (2019-02-11) Closed issue.
 - commit: 392e040647403fc8007708d52ce01d915b014849
-  note:
+  note: |
+    (2019-02-11) Removes l10n, which means that locale-dictated format has higher
+    precedence, for the coefficient if the Decimal has more than 200 digits. This
```

Comment:
I know what l10n is, but devs who don't aren't uncommon.  I would probably change this to "Localization (l10n)" And then just use l10n from there out